src/main/java/pl/debuguj/parkingspacessystem/exceptions/CarRegisteredInSystemException.java

```diff
@@ -0,0 +1,17 @@
+package pl.debuguj.parkingspacessystem.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+/**
+ * Created by grzesiek on 17.10.17.
+ */
+@ResponseStatus(HttpStatus.BAD_REQUEST)
```

Comment:
Nie jestem pewien czy to jest BAD REQUEST - request jest dobry, ale operacja jest niedozwolona